pkg/lib/bundle/build_test.go

```diff
@@ -50,7 +50,7 @@ func TestBuildBundleImage(t *testing.T) {
 
 	for _, item := range tests {
 		var cmd *exec.Cmd
-		cmd, err := BuildBundleImage(item.directory, item.imageTag, item.imageBuilder)
```

Comment:
Could the directory field be removed from the struct if unused?

Reply:
+1